HIRS_Utils/src/main/java/hirs/tpm/TPMBaselineGenerator.java

```diff
@@ -591,16 +591,15 @@ private void parseBaseline(final String baselineName, final InputStream inStream
             throws IOException, TPMBaselineGeneratorException {
         BufferedReader reader = new BufferedReader(new InputStreamReader(
                 inStream, "UTF8"));
-        String dataRow = reader.readLine();
+        String dataRow = "";
         HashMap<TPMBaselineFields, String> fieldMap = new HashMap<>();
 
         try {
-            while (reader.ready()) {
-                if (StringUtils.isBlank(dataRow)) {
-                    dataRow = reader.readLine();
-                    continue;
+            while (true) {
+                dataRow = reader.readLine();
+                if (dataRow == null || !dataRow.matches("[0-9a-zA-Z./()_,\" -]+")) {
```

Comment:
Since this regex is used here and a few lines below, can we abstract them both into a `private static final String` up at the top of the class. Not quite sure what to call it. Maybe just `VALID_BASELINE_REGEX`?